tests.php

```diff
@@ -0,0 +1,114 @@
+<?php
+	/* Tests script for phpcs-security-audit */
+
+	//eval($_GET['a']);
+	echo "aaaa";
+	echo "bbbb" . $_POST['b'];
+	echo "b";
+	db_query($_GET['a']);
+	preg_replace("/.*/ei", 'aaaaaaa', 'bbbbb');
+	preg_replace("/.*/ei", $_GET['a'], 'aaaaaaa');
+	preg_replace($_GET['b'], $_GET['a'], $_GET['c']);
+	preg_replace($b, $_GET['a'], 'aaaaaa');
+	preg_replace("aaa", $_GET['a'], 'ababaaa');
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> warning - Weird usage of preg_replace, please check manually for /e modifier.